.eslintrc

```diff
@@ -0,0 +1,3 @@
+{
+    "extends": "airbnb"
+}
```

Comment:
"parser": "babel-eslint"

Reply:
I still had some underlines and found some working config on web.

Reply:
Additionally I've found some eslint.config on the web, it has fewer issues, but still some underlines. 
I wondering is it there really right eslint.config for React?

Reply:
This is the rule requirements, so the options are: A) change extensions for each file with JSX, B) configure ESLint, refer to https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-filename-extension.md#rule-options